### PR TITLE
Language Added

### PR DESCRIPTION
**Pull request by TheUltimateDuel**
This is a small fix for a issue I had. It isn't well made but it works.

### CODE REVIEW

**Comment by builder-247** [comment]
Thank you for your PR!

I think it would make more sense to always change the language to english on autotip startup and then change it back to the preferred language in `gracefulShutdown`

File: index.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ function getUUID() {
   return bot._client.session.selectedProfile.id;
 }
 
+function setlang() {
+  logger.info('Changing language.');
+  bot._client.write('chat', { message: '/lang ' + config.CHANGE_LANGUAGE });
```
Comment (builder-247) [inline index.js:30]: You can just use `bot.chat()` here

Comment (TheUltimateDuel) [reply index.js:30]: I see, I was attempting to make it somewhat resemble the format of the code.

**Comment by TheUltimateDuel** [comment]
> Thank you for your PR!
> 
> I think it would make more sense to always change the language to english on autotip startup and then change it back to the preferred language in `gracefulShutdown`

I was wondering about how to do that but I couldn't figure out a way to add that

**Pull request merged**